pipeline-scripts/openshiftv4_on_azure.groovy

```diff
@@ -0,0 +1,150 @@
+			def openshift_install_pull_secret = openshiftv4_properties['OPENSHIFT_INSTALL_PULL_SECRET']
+			def openshift_install_quay_registry_token = openshiftv4_properties['OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN']
+			def openshift_install_image_registry = openshiftv4_properties['OPENSHIFT_INSTALL_IMAGE_REGISTRY']
+			def openshift_install_registry_token = openshiftv4_properties['OPENSHIFT_INSTALL_REGISTRY_TOKEN']
+			def openshift_install_installer_from_source = openshiftv4_properties['OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE']
+			def openshift_install_installer_from_source_version = openshiftv4_properties['OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE_VERSION']
+			def gopath = openshiftv4_properties['GOPATH']
+			def azure_subscription_id = openshiftv4_properties['AZURE_SUBSCRIPTION_ID']
+			def azure_tenant_id = openshiftv4_properties['AZURE_TENANT_ID']
+			def azure_service_principal_client_id = openshiftv4_properties['AZURE_SERVICE_PRINCIPAL_CLIENT_ID']
+			def azure_service_principal_client_secret = openshiftv4_properties['AZURE_SERVICE_PRINCIPAL_CLIENT_SECRET']
+                        def azure_base_domain_resource_group_name = openshiftv4_properties['AZURE_BASE_DOMAIN_RESOURCE_GROUP_NAME']
+			def azure_region = openshiftv4_properties['AZURE_REGION']
+			def openshift_base_domain = openshiftv4_properties['OPENSHIFT_BASE_DOMAIN']
+			def openshift_cluster_name = openshiftv4_properties['OPENSHIFT_CLUSTER_NAME']
+			def openshift_master_count = openshiftv4_properties['OPENSHIFT_MASTER_COUNT']
+			def openshift_worker_count = openshiftv4_properties['OPENSHIFT_WORKER_COUNT']
+			def openshift_master_vm_size = openshiftv4_properties['OPENSHIFT_MASTER_VM_SIZE']
+			def openshift_worker_vm_size = openshiftv4_properties['OPENSHIFT_WORKER_VM_SIZE']
+			def openshift_master_root_volume_size = openshiftv4_properties['OPENSHIFT_MASTER_ROOT_VOLUME_SIZE']
+			def openshift_worker_root_volume_size = openshiftv4_properties['OPENSHIFT_WORKER_ROOT_VOLUME_SIZE']
+			def openshift_cidr = openshiftv4_properties['OPENSHIFT_CIDR']
+			def openshift_machine_cidr = openshiftv4_properties['OPENSHIFT_MACHINE_CIDR']
+			def openshift_service_network = openshiftv4_properties['OPENSHIFT_SERVICE_NETWORK']
+			def openshift_host_prefix = openshiftv4_properties['OPENSHIFT_HOST_PREFIX']
+                        def openshift_post_install_poll_attempts = openshiftv4_properties['OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS']
+                        def openshift_toggle_infra_node = openshiftv4_properties['OPENSHIFT_TOGGLE_INFRA_NODE']
+                        def openshift_toggle_workload_node = openshiftv4_properties['OPENSHIFT_TOGGLE_WORKLOAD_NODE']
+                        def machineset_metadata_label_prefix = openshiftv4_properties['MACHINESET_METADATA_LABEL_PREFIX']
+                        def openshift_infra_node_instance_type = openshiftv4_properties['OPENSHIFT_INFRA_NODE_INSTANCE_TYPE']
```

Comment:
Once https://github.com/openshift-scale/scale-ci-deploy/pull/30 is merged this will change to `OPENSHIFT_INFRA_NODE_VM_SIZE`